src/commands/group/addadmin.ts

```diff
@@ -52,6 +52,9 @@ export default class AddAdmin extends Command {
             .group.addAdmins(groupID, flags.users as string[])
             .then((addResult) => {
                 this.log(`\n${this.buildResultTable(addResult).toString()}\n`);
+                if (addResult.failed.length > 0) {
+                    this.error(`Failed to add ${addResult.failed.length} admin(s).`);
```

Comment:
Should we tell this which admin failed?

Reply:
We already do. On line 54 we're displaying a table that shows the list of success and failures. This added log was just to cause that error to be obvious and exit with a non-zero code.